examples/radio/radio-2/js/radioGroupActiveDescendant.js

```diff
@@ -28,6 +28,7 @@ var RadioGroup = function (domNode) {
 
   this.keyCode = Object.freeze({
     'TAB': 9,
+    'RETURN': 13,
```

Comment:
Instead of adding here, I believe we need to remove the case for return from line 128 (now 129 in this file).

The table of supported keys does not include return or enter. The interaction pattern does not include enter or return. In this way, radios are similar to checkbox; enter does nothing. This should align with what native HTML radios do in most browsers.

Reply:
@mcking65 I can fix it